apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PageLinkBuilder.java

```diff
@@ -28,36 +28,17 @@
   private final String resource;
   private final String searchByDesc;
   private final String identifier;
-  private final DateRangeParam lastUpdated;
-  private final String excludeSamhsa;
-  private final Set<ClaimType> claimTypes;
+  private final RequestDetails requestDetails;
 
   public PageLinkBuilder(
-      RequestDetails requestDetails,
-      String resource,
-      String searchByDesc,
-      String identifier,
-      DateRangeParam lastUpdated,
-      Set<ClaimType> claimTypes,
-      String excludeSamhsa) {
+      RequestDetails requestDetails, String resource, String searchByDesc, String identifier) {
     this.pageSize = parseIntegerParameters(requestDetails, Constants.PARAM_COUNT);
     this.startIndex = parseIntegerParameters(requestDetails, "startIndex");
     this.serverBase = requestDetails.getServerBaseForRequest();
     this.resource = resource;
     this.searchByDesc = searchByDesc;
     this.identifier = identifier;
```

Comment:
It looks like these can be removed, I don't see them being used anywhere.

Reply:
Good catch. Will verify and remove them.